game_tic_tac_toe.py

```diff
@@ -159,11 +184,130 @@ def player_won(self, check_disable):
                         {0, 3, 6}, {1, 4, 7}, {2, 5, 8},
                         {2, 4, 6}, {0, 4, 8}]
         for win_set in win_combinations:
-            if win_set.issubset(self.check_disable):
-                for elem in win_set:
-                    self.game_btns[elem]['bg'] = '#CCFFFF'
-                return True
-
+            if win_set.issubset(check_disable):
+                if game_state == 'playing':
+                    for elem in win_set:
+                        self.game_btns[elem]['bg'] = '#CCFFFF'
+                    return True
+                elif game_state == 'trying':
+                    return True
```

Comment:
Just replace by one `return True` which is indented to be inside the `if win_set.issubset(check_disable):` block. So you don't need the extra `elif`.